File: Web/EYE/EYE/Model/Criptografia.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Security.Cryptography;
using System.Text;
using System.Web;

namespace EYE.Model
{
    public class Criptografia
    {
        public static string GerarSenhaHash(string senha, int salt)
        {
            MD5 md5Hash = MD5.Create();
            byte[] data = md5Hash.ComputeHash(Encoding.UTF8.GetBytes($"{senha}{salt}"));

            StringBuilder sBuilder = new StringBuilder();
            for (int i = 0; i < data.Length; i++)
            {
                sBuilder.Append(data[i].ToString("x2"));
            }
            return sBuilder.ToString();
        }

        public static int GerarSalt()
        {
            return new Random().Next();
        }
    }
}

File: Local/Eye/src/main/java/br/com/eye/model/Criptografia.java
package br.com.eye.model;

import java.math.BigInteger;
import java.security.MessageDigest;
import java.security.NoSuchAlgorithmException;

public class Criptografia {

    public static String gerarSenhaHash(String senha, int salt) throws NoSuchAlgorithmException {
        String senhaComposta = senha + salt;
        MessageDigest md = MessageDigest.getInstance("MD5");
        md.update(senhaComposta.getBytes());
        byte[] senhaByte = md.digest();
        StringBuilder s = new StringBuilder();
        for (int i = 0; i < senhaByte.length; i++) {
            int parteAlta = ((senhaByte[i] >> 4) & 0xf) << 4;
            int parteBaixa = senhaByte[i] & 0xf;
            if (parteAlta == 0) {
                s.append('0');
            }
            s.append(Integer.toHexString(parteAlta | parteBaixa));
        }
        return s.toString();

    }
}
